### ISSUE TITLE
rename `pcoa` parameter name?

### ISSUE THREAD

**Issue by sjanssen2** [qiime2/q2-emperor#95]
We have introduced `t-SNE` and `UMAP` as alternative ordination methods in qiime2-2021.11. Should we therefore change the input parameter name for the ordination (which previously always was a PCoA) from "pcoa" into "ordination"?

https://github.com/qiime2/q2-emperor/blob/fdd1a8b56f93f161d0ca24630fcaec798a6ecf1c/q2_emperor/plugin_setup.py#L65

**Comment by ElDeveloper** [comment]
I agree with this! Maybe the way to do this is to first create a new type that's named "OrdinationResults" not "PCoAResults". Then we can add that type to the relevant actions and rename where appropriate. @thermokarst @ebolyen any suggestions on how to do this with backwards compatibility in mind?
